source/utilities.tex

```diff
@@ -21723,7 +21723,7 @@
 \pnum
 \returns
 \tcode{x.size() <=> y.size()} if \tcode{x.size() != y.size()};
-\tcode{lexicographical_compare_3way(\newline
+\tcode{lexicographical_compare_three_way(\newline
```

Comment:
The manual `\newline` needs to go, otherwise we get a bad line break.

```suggestion
\tcode{lexicographical_compare_three_way(
```

Reply:
Done

Reply:
Did you actually build the PDF with that change? Now there is an extra space after the `(`, so some more fixes are in order.

Further, your name shows up as

`Author: lam-work-group <77063444+lam-work-group@users.noreply.github.com>`

in the resulting git history. Is that how you wish to be cited?